contracts/libs/0.8.x/BytecodeStorageV1.sol

```diff
@@ -46,7 +88,7 @@ library BytecodeStorage {
     /**
      * @notice Write a string to contract bytecode
      * @param _data string to be written to contract. No input validation is performed on this parameter.
-     * @return address_ address of deployed contract with bytecode containing concat(invalid opcode, deployer-address, data)
+     * @return address_ address of deployed contract with bytecode containing
```

Comment:
Should we remove the `containing` from this comment?

Reply:
Ah good catch – will fix and resolve this comment once I've done so.

Reply:
Updated!